### PR TITLE
Version 0.2.3 dependencies fixed for case sensitive

### PR DESCRIPTION
**Pull request by ppanero**
Case sensitive installations

### CODE REVIEW

**Comment by bastelfreak** [comment]
Hi @ppanero, thanks for the PR. Can you squash your commits into one commit?

**Comment by ppanero** [comment]
@bastelfreak sorry didnot notice that. Done

File: metadata.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "puppet-misp",
-  "version": "0.2.2",
+  "version": "0.2.3-rc0",
```
Comment (bastelfreak) [inline metadata.json]: it seems like you rebased against an older version of master. Can you rebase again, against the latest version?

**Comment by ppanero** [comment]
@bastelfreak Done, sorry for the mess :( learning by doing I guess...

**Comment by bastelfreak** [comment]
Don't worry, we all started like this. Can you please take a look at the used email address? It isn't associated with your github account.

**Comment by ppanero** [comment]
@bastelfreak  I added the official one, but I did it from home now and its using some @visitor.... Ill take a lot at changing that. But could we merge this cuz we kinda need it for a demo tomorrow? plz :D

**Comment by bastelfreak** [comment]
haha :D. I know this very well. Thats why I use my personal email address, even for work related commits.

**Comment by ppanero** [comment]
@bastelfreak Thanks a lot will take it into account for the next ones :D

**Comment by ppanero** [comment]
@bastelfreak sorry for bothering again, could you run the rake travis_release script so latest is in puppetforge? I know is a minor but it makes some machines fail :D

**Pull request merged**